src/aasthaJJun20/ValidateMobileNumber.java

```diff
@@ -0,0 +1,67 @@
+package aasthaJJun20;
+
+//Assignment-24 - Validate given phone number 
+public class ValidateMobileNumber {
+	// below method checks if there is any alphabet or special char in given number
+	private boolean isNumber(String number) {
+		for (int index = 0; index < number.length(); index++) {
+			if (!Character.isDigit(number.charAt(index)))
+				return false;
+		}
+		return true;
+	}
+
+	// below method checks if the given number starts with 9/8/7
+	private boolean checkFirstDigit(char ch) {
+		if (ch == '9' || ch == '8' || ch == '7')
+			return true;
+		return false;
+	}
+
+	public void verifyMobileNumber(String number) {
+		switch (number.length()) {
+		// below case - is to validate numbers like : +91 9765463742
+		case 14:
+			if ((number.startsWith("+91 ")) && (checkFirstDigit(number.charAt(4))) && isNumber(number.substring(4)))
+				System.out.println(number + " --> Valid");
```

Comment:
One suggestion: By using substring, the input string is sanitized (i.e. getting rid of  + here) and then given to isNumber method. We need to give different substrings for different types of Ph No strings also, we need to remove spaces in another format. This is logically absolutely fine but, I would recommend we do this sanitization in the method and ask user to pass just the string he has...